### DISCUSSION TITLE
👋 Welcome to the Skyhook Data Management Discussions

### DISCUSSION THREAD

**Discussion by carlosmalt** [skyhookdm/.github#2] [Announcements]
Github Discussions are now enabled for Skyhook. The goal is to provide a more structured forum for Skyhook-related discussions than the single #skyhook Slack channel. 

Our [NSF POSE Phase 1 funding](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2229773) allows us to scope out how to create a Skyhook open source ecosystem (OSE), how to sustainably support it, and how to enable growth of a diverse community around it, with a long-term vision aimed at developing OSEs for other research products to increase impact and innovation.

**Comment by drin** [comment]
cool, the discussions interface seems interesting so far. Maybe for now we can try and hook up discussions to a slack channel so that people there can easily see when new discussions (discussion topics?) are created. As of now, I only happened to see these when I was looking at the org page and saw "Top discussions..." on the right sidebar

**Reply by carlosmalt** [reply]
Thanks @drin -- I've subscribed #skyhook to skyhookdm. Let's see whether this comment shows up.

**Reply by carlosmalt** [reply]
First attempt didn't work, needed to install slack app on skyhookdm. Enabled all repositories. Let's see whether this shows up ...

**Reply by carlosmalt** [reply]
Looks like replies are not propagated 😖

**Comment by carlosmalt** [comment]
This is to find out whether comments are propagated to subscribed slack channels...

**Reply by carlosmalt** [reply]
Nope.
